.gitguardian.yml

```diff
@@ -0,0 +1,3 @@
+secret:
+  ignored-paths:
+    - 'data/backend/authentication/migratons/*'
```

Comment:
It's `migrations`, not `migratons`

Reply:
Also, according to [the official docs](https://docs.gitguardian.com/ggshield-docs/configuration#configuration-files), it's `ignored_paths`, not `ignored-paths` (we want an underscore instead of an hyphen)

![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/ea48fb8a-cbf2-4c56-ac46-d887f55bacff)